src/pages/ScholarshipDetailsPage.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
Interesting, so the accordion would be used by all scholarships or just those with little details?

Reply:
If the details for a scholarship don't take up much space, I can see it being used for all scholarships. Otherwise I can imagine it being used just to show the most essential information regarding a scholarship.